src/Nowruz/modules/Jobs/containers/OrgOfferModal/index.tsx

```diff
@@ -105,12 +109,12 @@ export const OrgOfferModal: React.FC<OrgOfferModalProps> = ({ open, onClose, app
                 <div className={css.row}>
                   {isCrypto && renderfieldInfo('Your wallet', 'Connect wallet to send an offer')}
                   {isCrypto && (
-                    <div className="flex justify-center my-5">
+                    <div className="flex justify-center my-5 z-30">
                       <Dapp.Connect />

```

Comment:
Would you mind to take a look at PR#374, PaymentCrypto component? you can use this code: 
 `{isConnected ? <Dapp.Connect /> : <ConnectButton handleClick={() => openConnect()} />}`
 to have the button with the new design

Reply:
I saw the exchange of conversation. Would you like me to do it as a separate PR? And I need to wait for you PR to get merge so I can apply this.